bitnami/kafka/README.md

```diff
@@ -0,0 +1,215 @@
+| `auth.enabled`                   | Switch to enable the kafka authentication.                                                                 | `false`                                                            |
+| `auth.existingSecret`            | Name of the existing secret containing credentials for brokerUser, interBrokerUser and zookeeperUser.      | `nil`                                                              |
+| `auth.certificatesSecret`        | Name of the existing secret containing the certificate files that will be used by Kafka.                   | `nil`                                                              |
+| `auth.certificatesPassword`      | Password for the above certificates if they are password protected.                                        | `nil`                                                              |
+| `auth.brokerUser`                | Kafka client user.                                                                                         | `user`                                                             |
+| `auth.brokerPassword`            | Kafka client password.                                                                                     | `nil`                                                              |
+| `auth.interBrokerUser`           | Kafka inter broker communication user                                                                      | `admin`                                                            |
+| `auth.interBrokerPassword`       | Kafka inter broker communication password.                                                                 | `nil`                                                              |
+| `auth.zookeeperUser`             | Kafka Zookeeper user.                                                                                      | `nil`                                                              |
+| `auth.zookeeperPassword`         | Kafka Zookeeper password.                                                                                  | `nil`                                                              |
+| `securityContext.enabled`        | Enable security context (redis master pod)                                                                 | `true`                                                             |
+| `securityContext.fsGroup`        | Group ID for the container (redis master pod)                                                              | `1001`                                                             |
+| `securityContext.runAsUser`      | User ID for the container (redis master pod)                                                               | `1001`                                                             |
```

Comment:
Remove redis master pod in the three lines above

Reply:
done